src/core/BufferGeometry.js

```diff
@@ -50,6 +49,13 @@ THREE.BufferGeometry.prototype = {
 
 	},
 
+	offsets: function () {
+
+		console.warn( 'THREE.BufferGeometry: .offsets has been renamed to .drawcalls.' );
+		return this.drawcalls.slice();
```

Comment:
I see... I think I would just return `this.drawcalls`. Seems like it will be less destructive.